### ISSUE TITLE
Project Needs a Better Name

### ISSUE THREAD

**Issue by machineghost** [arijs/prettier-with-tabs#34]
First off let me just say that I am so glad you started this project, as I think there is a real need for Prettier without the baggage of the Prettier philosophy (ie. the philosophy that configuration options are bad and if you leave them out programmers everywhere will magically join hands, sing songs, and format their code the exact same way).

That being said, the current library name doesn't really communicate that its the "better, more customizable" code formatter at all.  With all due respect,  it sounds more like a Prettier plug-in or sub-package, not even a code formatter in its own right.

Might I propose instead (any of which could optionally have "Formatter" or "Code Formatter" at the end):

* Prettier Your Way
* Pretty Customizable
* Prettier Customizable
* Prettier Configurable
* Prettier With Customization
* Philosophy-Free Prettier
* Prettier Than Will Smith
* Prettier for Real Teams
* Fully Customizable Code Formatter
* Custom Javascript Formatter
* The People's Javascript Formatter

and my personal favorite:

* Prettier Than Prettier

Or if you don't like those, switching to any other name other which communicates either "like Prettier, but better", "like Prettier, but more configurable", or "a configurable code formatter" (or a combination) would be an improvement.

This may not seem like it matters, but when someone is deciding what code formatter to use for the first time the name makes a huge impact, and we want to get people using your library and not Prettier core right?

**Comment by iakovmarkov** [comment]
I agree on the fact that project name is a bit misleading - I've thought that this is some set of plugins for prettier or something like this.

TBH I don't like any of the ideas that you proposed because they are long, and that's IMO bad for library names. What about **prettiest**? It implies that it's better that prettier because of superlative form, and maintains the connection to the original lib.

**Comment by brody2consult** [comment]
I published a new [`prettierx`](https://www.npmjs.com/package/prettierx) fork in case it may help anyone.
